discord/installer/lib/paths.js

```diff
@@ -34,6 +34,8 @@ function getAddonExtractPath() {
             return path.join(process.env.APPDATA, 'discord');
         case 'darwin':
             return path.join(process.env.HOME, '/Library/Preferences/discord');
+        case 'linux':
+            return path.join(process.argv[3]);
```

Comment:
Currently we use the 2nd command line argument to tell the install script whether or not we're installing to the PTB.  While I realize that Linux is a PTB by its very nature right now we should have something along the lines of an actual flag in our arguments to pass this in.  I'll see what I can do to help support this as the arguments in the installer are a bit of a hack right now anyway.